R/bs-dependencies.R

```diff
@@ -111,7 +111,7 @@ bs_theme_dependencies <- function(
     warning("Failed to copy over bootstrap's javascript files into the htmlDependency() directory.")
   }
 
-  c(
+  htmltools::resolveDependencies(c(
```

Comment:
Why is this changed to `resolveDependencies`?

Reply:
IIRC, it was to handle the new `tagFunction()` that comes with `bs_font_dependencies()`, but come to think of it, maybe `setCurrentTheme()` should be resolving dependencies?

Reply:
After thinking about this a bit more, this seems fine for now, but we should probably do this inside `setCurrentTheme()` since, in some edge cases, folks might want to call `bs_theme_dependencies()` directly without resolving the dependencies straight away.